sketch_map_tool/templates/help.html

```diff
@@ -64,23 +70,61 @@ <h1 id="step-by-step">{{ _('Step-by-Step Instructions') }}</h1>
                             represents the
                             boundaries of your area of interest, which will be printed on the Sketch Map.') }}</li>
-                        <li>{{ _('Initiate map creation by clicking the blue "next" button.') }}</li>
+                        <li>{{ _('Initiate the map creation by clicking the blue "next" button.') }}</li>
                     </ol>
                 
                     <figure> <!-- figure needs to be changed. link to the figure-->
                         <img src="{{ url_for("static", filename="assets/images/help/1_1_AOI.webp" ) }}"
                             alt="{{ _('Screenshot of the Sketch Map Tool.') }}">
                         <figcaption></figcaption>
                     </figure>
-                
+                    <p>
+                        {{ _('<strong>Searching by coordinates</strong>') }}
+                    </p>                
+                    <p>
+                        {{ _('Using coordinates to search for an accurate location is particularly useful in rural areas. 
+                        Ensure that the coordinates are in the coordinate system WGS 84 reference system, expressed as latitude and longitude (LAT and LON).') }}
+                    </p>
+                    <p>
+                        {{ _('Example: We have the following coordinates of Nairobi in the WGS 84 coordinate reference system:  1.2921° S (latitude) and 36.8219° E (longitude). 
+                        In the search field just type LAT, LON like shown below:') }}
+                    </p>
+
+                    <figure>
+                        <img src="{{ url_for("static", filename="assets/images/help/1_1_searching_by_coordinates.webp" ) }}"
+                            alt="{{ _('Screenshot of the coordinates.') }}">
+                        <figcaption></figcaption>
+                    </figure>
+                    
+                    <p>
+                        {{ _('Be sure to include the hemisphere information (South/North, West/East) to avoid incorrect localization.') }}
+                    </p>
+                    <p>
+                        {{ _('If you need to transform your available coordinates into Lat-Lon coordinates first, we can recommend you to use this tool: 
+                            <a href="https://coordinates-converter.com/en/decimal/51.000000,10.000000?karte=OpenStreetMap&zoom=8" 
+                            target="_blank">Online converter to all coordinate systems</a>') }}
+                    </p>
+                    <p>
+                        {{ _('The marking of your searched point that appears during the map selection, will not be visible on the printed map. 
+                        If you need this point as a point of reference on your paper Sketch Maps please mark it after you downloaded the pdf 
+                        (either manually with a pen or digitally e.g. in a photo editing program like GIMP).') }}
```

Comment:
Not sure if I would recommend `GIMP` (a raster image edit program) as a program to edit PDFs. Can't basically every better PDF program set a pointer or marking on a PDF?